source/numerics.tex

```diff
@@ -3869,8 +3869,8 @@
  nondeterministic uniform random bit generator object.
  The semantics and default value of the \tcode{token}
  parameter are
- \impldef{semantics and default value of \tcode{token} parameter to \tcode{random_device} constructor}.
- \footnote{The parameter is intended
+ \impldef{semantics and default value of \tcode{token} parameter to \tcode{random_device} constructor}.%
+\footnote{The parameter is intended
```

Comment:
Yarkhs, why is there a weird blank column at the start here... never mind, that can be revisited at a later point.